app/javascript/dashboard/components/layout/AvailabilityStatus.vue

```diff
@@ -81,11 +53,17 @@ export default {
       const availabilityIndex = AVAILABILITY_STATUS_KEYS.findIndex(
         key => key === this.currentUserAvailabilityStatus
       );
+      console.log(
+        this.$t('PROFILE_SETTINGS.FORM.AVAILABILITY.STATUSES_LIST')[
+          availabilityIndex
+        ]
+      );
       return this.$t('PROFILE_SETTINGS.FORM.AVAILABILITY.STATUSES_LIST')[
         availabilityIndex
       ];
     },
     currentUserAvailabilityStatus() {
+      console.log(this.currentUser.availability_status);
```

Comment:
Can you remove this line `console.log(this.currentUser.availability_status);`

Reply:
```suggestion
```